events/wc_primitive_events.txt

```diff
@@ -789,3 +793,166 @@ character_event = {
 		name = EVTOPTA_WCPRM_605
 	}
 }
+
+
+character_event = {
+    id = WCPRM.500
+    picture = GFX_evt_kobold_statute
+    desc = EVTDESC_WCPRM_500
+    border = GFX_event_normal_frame_religion
+	
+    is_triggered_only = yes
+	
+    immediate = {
+	    random_list = {
+		    20 = {
+			    character_event = {
+				    id = WCPRM.501 days = 7
+			    }
+		    }
+		    30 = {
+			    character_event = {
+			        id = WCPRM.502 days = 7
+			    }
+		    }
+			50 = {
+			    character_event = {
+				    id = WCPRM.503 days = 7
+				}
+			}
+	    }
+    }
+	
+	option = {
+	    name = EVTOPTA_WCPRM_500
+		piety = 1
+		wealth = -10
```

Comment:
If you have no other options, but pay. What if you don't have money?

Reply:
I will add other option to say no